plugins/package-managers/conan/src/main/kotlin/Conan.kt

```diff
@@ -130,7 +130,7 @@ class Conan(
     override val globsForDefinitionFiles = listOf("conanfile*.txt", "conanfile*.py")
 
     private val handler by lazy {
-        if (command.getVersion().startsWith("1.")) {
+        if (command.getVersion().startsWith("Conan version 1.")) {
```

Comment:
Removing this prefix is already done in line 77, so I don't think it's necessary.

Reply:
Maybe @nnobelis can help here. But please move the discussion to, well, GitHub [discussions](https://github.com/oss-review-toolkit/ort/discussions) or to [Slack](http://slack.oss-review-toolkit.org/) then.

Reply:
Makes sense. Sorry for the noise